modules/@apostrophecms/asset/index.js

```diff
@@ -396,6 +397,16 @@ module.exports = {
             };
 
             components.forEach((component, i) => {
+              if (options.requireDefaultExport) {
+                if (!fs.readFileSync(component, 'utf8').match(/export[\s\n]+default/)) {
```

Comment:
Would there be no way to use a named export in a front end file?

Reply:
Not in a `ui/src/index.js` no, those are for Apostrophe to import, if you want to provide code for reuse by various modules you can, you just wouldn't put it in `ui/src/index.js` and then you can import to your heart's content, use named exports, etc.